Log message:

Make sun3 scsi drivers compile/work again (though with way too many warnings...)

Tested on 3/50, 3/60.

Modified files:
    linux/drivers/scsi:
	Kconfig sun3_NCR5380.c sun3_scsi.c sun3_scsi.h sun3_scsi_vme.c 


Index: linux/drivers/scsi/Kconfig
Stats: 2 modifications
http://linux-m68k-cvs.ubb.ca/c/cvsweb/linux/drivers/scsi/Kconfig.diff?r1=1.20&r2=1.21
========================================================================
--- linux/drivers/scsi/Kconfig	9 Apr 2006 22:23:01 -0000	1.20
+++ linux/drivers/scsi/Kconfig	11 May 2006 11:16:27 -0000	1.21
@@ -1785,7 +1785,7 @@
 
 config SUN3_SCSI
 	tristate "Sun3 NCR5380 SCSI"
-	depends on SUN3 && SCSI && BROKEN
+	depends on SUN3 && SCSI
 	select SCSI_SPI_ATTRS
 	help
 	  This option will enable support for the OBIO (onboard io) NCR5380
Index: linux/drivers/scsi/sun3_NCR5380.c
Stats: 4 modifications
http://linux-m68k-cvs.ubb.ca/c/cvsweb/linux/drivers/scsi/sun3_NCR5380%2ec.diff?r1=1.1.1.16&r2=1.25
========================================================================
--- linux/drivers/scsi/sun3_NCR5380.c	9 Apr 2006 16:55:01 -0000	1.1.1.16
+++ linux/drivers/scsi/sun3_NCR5380.c	11 May 2006 11:16:27 -0000	1.25
@@ -1269,7 +1269,7 @@
 	NCR_PRINT(NDEBUG_INTR);
 	if ((NCR5380_read(STATUS_REG) & (SR_SEL|SR_IO)) == (SR_SEL|SR_IO)) {
 	    done = 0;
-	    ENABLE_IRQ();
+//	    ENABLE_IRQ();
 	    INT_PRINTK("scsi%d: SEL interrupt\n", HOSTNO);
 	    NCR5380_reselect(instance);
 	    (void) NCR5380_read(RESET_PARITY_INTERRUPT_REG);
@@ -1302,7 +1302,7 @@
 		INT_PRINTK("scsi%d: PHASE MISM or EOP interrupt\n", HOSTNO);
 		NCR5380_dma_complete( instance );
 		done = 0;
-		ENABLE_IRQ();
+//		ENABLE_IRQ();
 	    } else
 #endif /* REAL_DMA */
 	    {
Index: linux/drivers/scsi/sun3_scsi.c
Stats: 1 insertion, 1 deletion
http://linux-m68k-cvs.ubb.ca/c/cvsweb/linux/drivers/scsi/sun3_scsi%2ec.diff?r1=1.1.1.17&r2=1.19
========================================================================
--- linux/drivers/scsi/sun3_scsi.c	14 Jan 2006 23:13:16 -0000	1.1.1.17
+++ linux/drivers/scsi/sun3_scsi.c	11 May 2006 11:16:27 -0000	1.19
@@ -75,9 +75,9 @@
 #define REAL_DMA
 
 #include "scsi.h"
+#include "initio.h"
 #include <scsi/scsi_host.h>
 #include "sun3_scsi.h"
-#include "NCR5380.h"
 
 static void NCR5380_print(struct Scsi_Host *instance);
 
Index: linux/drivers/scsi/sun3_scsi.h
Stats: 2 modifications
http://linux-m68k-cvs.ubb.ca/c/cvsweb/linux/drivers/scsi/sun3_scsi%2eh.diff?r1=1.1.1.9&r2=1.13
========================================================================
--- linux/drivers/scsi/sun3_scsi.h	14 Jan 2006 23:13:16 -0000	1.1.1.9
+++ linux/drivers/scsi/sun3_scsi.h	11 May 2006 11:16:27 -0000	1.13
@@ -220,7 +220,7 @@
  *
  */
 
-
+#include "NCR5380.h"
 
 #if NDEBUG & NDEBUG_ARBITRATION
 #define ARB_PRINTK(format, args...) \
Index: linux/drivers/scsi/sun3_scsi_vme.c
Stats: 1 insertion, 1 deletion
http://linux-m68k-cvs.ubb.ca/c/cvsweb/linux/drivers/scsi/sun3_scsi_vme%2ec.diff?r1=1.1.1.10&r2=1.9
========================================================================
--- linux/drivers/scsi/sun3_scsi_vme.c	14 Jan 2006 23:13:16 -0000	1.1.1.10
+++ linux/drivers/scsi/sun3_scsi_vme.c	11 May 2006 11:16:27 -0000	1.9
@@ -41,9 +41,9 @@
 #define REAL_DMA
 
 #include "scsi.h"
+#include "initio.h"
 #include <scsi/scsi_host.h>
 #include "sun3_scsi.h"
-#include "NCR5380.h"
 
 extern int sun3_map_test(unsigned long, char *);
 
-
To unsubscribe from this list: send the line "unsubscribe linux-m68k-cvscommit" in
the body of a message to majordomo@vger.kernel.org
More majordomo info at  http://vger.kernel.org/majordomo-info.html

